java/arcs/core/storage/ProxyInterface.kt

```diff
@@ -29,11 +31,18 @@ sealed class ProxyMessage<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
         is Operations -> copy(id = id)
     }
 
+    fun withMuxId(muxId: String): ProxyMessage<Data, Op, ConsumerData> = when (this) {
```

Comment:
KDoc is required on all public methods.

Reply:
I've put a KDoc on both `withMuxId` and `withId`